learning_resources/etl/constants.py

```diff
@@ -106,3 +109,10 @@ class ContentTagCategory(ExtendedEnum):
     "Assignments": ContentTagCategory.problem_sets.value,
     # Can add more here if ever needed, in format tag_name:category
 }
+
+
+@dataclass
+class ResourceNextRunConfig:
+    next_start_date: datetime = None
+    prices: list[Decimal] = list
```

Comment:
Should use `dataclasses.field` to specify a `[]` default, this is currently assigning the function `list` as the default:
```suggestion
    prices: list[Decimal] = field(default_factory=list)
```

Reply:
done